src/utils/permutations.ts

```diff
@@ -0,0 +1,19 @@
+const permutations = <T>( sequence: Array<T> ): Array<Array<T>> => {
```

Comment:
would be nice to have some description with examples here

Reply:
fixed